ouroboros-consensus-shelley/src/Ouroboros/Consensus/Shelley/Ledger/Query.hs

```diff
@@ -544,16 +544,16 @@ encodeShelleyResult query = case query of
     GetFilteredDelegationsAndRewardAccounts {} -> toCBOR
     GetGenesisConfig                           -> toCBOR
     DebugNewEpochState                         -> toCBOR
-    DebugChainDepState                         -> toCBOR
+    DebugChainDepState                         -> encode
```

Comment:
I'm not sure why that change came, tbh!

Reply:
It's not a one-liner, so we're going to accept this slight backwards progress in the `Serialise`-vs-`ToCBOR` tug of war because this PR is already so large.